classes/managers/manage_items.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../book'
+
+class ManageItems
+  def initialize(items = [])
+    @items = items
+  end
+
+  def carry_out(action)
+    types = { 1 => 'Book', 2 => 'Game', 3 => 'Music Album' }
+    puts "What do you want to #{action}? Chose by number"
+    types.each { |key, value| puts "#{key} - #{value}" }
+    choice = gets.chomp.to_i
+    method_from_type = types[choice].downcase.split.join('_')
+    send("#{action}_#{method_from_type}")
+  end
+
+  def list_items
+    carry_out('list')
+  end
+
+  def add_item
+    carry_out('add')
+  end
+
+  def list_book
+    @items.each do |item|
+      puts "#{item.class} - published at #{item.publish_date} by #{item.publisher} and the cover is #{item.cover_state}"
+    end
+  end
```

Comment:
- Please make sure that the way you save `item.publisher` and the way you access it, are the same.
![image](https://user-images.githubusercontent.com/43172164/182633716-27ee6809-999e-40f5-bee5-79c05c47b210.png)